poetry.lock

```diff
@@ -592,6 +592,14 @@ python-versions = ">=2.6, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
 [package.extras]
 test = ["ipaddress", "mock", "unittest2", "enum34", "pywin32", "wmi"]
 
+[[package]]
```

Comment:
I'm not sure why the poetry lock file was updated here as there have been no updates to dependencies. Also the trimesh updated version now doesn't match the stored hashes which I presume will cause errors when attempting a poetry install.